x/oracle/genesis.go

```diff
@@ -2,6 +2,7 @@ package oracle
 
 import (
 	"fmt"
+	"time"
 
```

Comment:
Could you add a genesis `export -> import` or `import -> export` test to the module?

I noticed we don't have one here like in the other modules